CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+- Cover new scenario of forbidden.
```

Comment:
It would be nice to add a few more details about this new scenario in case we need to find this PR in the future